src/main/java/com/devonfw/devcon/common/utils/Constants.java

```diff
@@ -43,6 +43,16 @@
    */
   public static final String GLOBAL_PARAMS_FILE = "globalParameters.json";
 
+  /**
+   * The name of the version parameters file
+   */
+  public static final String VERSION_PARAMS_FILE = "version.json";
+
+  /**
+   * The full path of the version parameters file
+   */
+  public static final String VERSION_PARAMS_FILE_FULL_PATH = "conf" + File.separator + VERSION_PARAMS_FILE;
+
```

Comment:
I think version.json file is missing in conf directory.

Reply:
Yes, you are right.  As I mentioned in my comments above, these changes suppose the file "version.json" is stored on the local folder "DevonServerPath/conf". Hopefully, this will happen in the next Devon release.